modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -133,6 +133,48 @@ using GRunArgP = util::variant<
     >;
 using GRunArgsP = std::vector<GRunArgP>;
 
+//GAPI_EXPORTS cv::GRunArgsP bind(cv::GRunArgs &results);
+inline cv::GRunArgsP bind(cv::GRunArgs &results)
+{
+    cv::GRunArgsP outputs;
+    outputs.resize(results.size());
+    unsigned int i = 0;
+    for (auto && res_obj : results)
+    {
+        //auto &res_obj = std::get<0>(it);
+        // FIXME: this conversion should be unified
+        using T = cv::GRunArg;
+        switch (res_obj.index())
+        {
+#if !defined(GAPI_STANDALONE)
+        case T::index_of<cv::UMat>() :
+        {
+            outputs[i] = &(cv::util::get<cv::UMat>(res_obj));
+        }
+                                     break;
```

Comment:
Something is wrong with this alignment..

Reply:
Done